code/src/main/java/com/onlinecontacttracing/storage/data-storage/PositiveUserContacts.java

```diff
@@ -31,8 +33,10 @@ public String getUserId() {
     return userId;
   }
 
-  public Set<PotentialContact> getListOfContacts() {
-    return listOfContacts;
+  public List<PotentialContact> getListOfContacts() {
+    List<PotentialContact> sortedContacts = new ArrayList(listOfContacts);
+    Collections.sort(sortedContacts);
```

Comment:
Please at least comment how this sorting is done.

Reply:
done